src/mngt/ocf_mngt_cache.c

```diff
@@ -2335,6 +2365,7 @@ struct ocf_pipeline_properties _ocf_mngt_cache_activate_pipeline_properties = {
 		OCF_PL_STEP(_ocf_mngt_test_volume),
 		OCF_PL_STEP(_ocf_mngt_init_promotion),
 		OCF_PL_STEP(_ocf_mngt_load_add_cores),
+		OCF_PL_STEP(_ocf_mngt_activate_cleaning_add_cores),
```

Comment:
During the regular recovery adding core to the cleaning policy is a step of reinitializing cleaning policy. It can be done because we have access to the valid superblock and info about added cores.
Missing info about valid cores is the reason why cores can't be added to the cleaning policy during the passive recovery and needs to be postponed to the activate.